Source/Eto.WinForms/Forms/Controls/DrawableHandler.cs

```diff
@@ -11,22 +11,23 @@ public class DrawableHandler : WindowsPanel<DrawableHandler.EtoDrawable, Drawabl
 	{
 		public virtual bool SupportsCreateGraphics { get { return true; } }
 
-		public class EtoDrawable : swf.Control
+		public class EtoDrawable : PanelBase<DrawableHandler>
```

Comment:
I see no way to review changes to this file just by looking at them. It has to be tested well... and it was, deeply ;)